README.md

```diff
@@ -44,9 +44,11 @@ This file contains a number of front-end interview questions that can be used wh
 * If you could master one technology this year, what would it be?
 * What are the differences between Long-Polling, Websockets and SSE?
 * Explain the importance of standards and standards bodies.
-* What is FOUC? How do you avoid FOUC?
+* What is Flash of Unstyled Content (FOUC)? How do you avoid FOUC?
 * Do your best to describe the process from the time you type in a website's URL to it finishing loading on your screen.
-* Explain what ARIA and screenreaders are, and how to make a website accessible.
+* What is your approach to accessiblity and developing in keyboard accessibility and visual focus, such as using foundational HTML and Javascript techniques? 
```

Comment:
This feels a little awkward to me - how about `What is your approach to accessibility with regards to keyboard control and visual focus using standard (non-ARIA) HTML and JavaScript techniques?`